### PR TITLE
HUB-813 Update Dockerfiles to use GHCR

### PR DESCRIPTION
**Pull request by j-maynard**

### CODE REVIEW

File: dockerfiles/aws-ruby/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.6.1
+FROM ghcr.io/alphagov/verify/ruby:2.6.6
 
 LABEL ruby="2.6.1"
```
Comment (galund) [inline dockerfiles/aws-ruby/Dockerfile:3]: Surely the label should match

**Review by galund — CHANGES_REQUESTED** [review]

File: dockerfiles/aws-ruby/Dockerfile (hunk 1 above)
Comment (CharlesIC) [reply dockerfiles/aws-ruby/Dockerfile:3]: That's the problem with having a vendoring pipeline - it's not great for having specific versions. We are planning to solve this

**Review by CharlesIC — APPROVED** [review]
We should check stuff works on updated versions of things

File: dockerfiles/aws-ruby/Dockerfile (hunk 1 above)
Comment (j-maynard) [reply dockerfiles/aws-ruby/Dockerfile:3]: @galund 2.6.6 is a minor point release and doesn't break this.  It also put this on par with all our other repositories which all use Ruby 2.6.6.

Comment (galund) [reply dockerfiles/aws-ruby/Dockerfile:3]: In which case the label should just be "2.6" - this makes no sense as is, and will just confuse us.

**Review by galund — APPROVED** [review]

**Pull request merged**